frontend/compositions/visualizations/map/BaseMap.tsx

```diff
@@ -16,101 +12,70 @@ import { FakeData } from "../utilities/chartTypes"
 
-  const valueScale = d3.scaleLinear().domain([0, 5]).range([0, 1])
-  const colorGradient = d3.interpolate("#7caed7", "#303463")
-
-  const svg = d3.select(baseMapRef.current)
+  const valueScale = scaleLinear().domain([0, 5]).range([0, 1])
+  const colorGradient = interpolate("#7caed7", "#303463")
 
+  const svg = select(baseMapRef.current)
   useEffect(() => {
-    if (!geoData) return
-
+    // if (!geoData) return
     // Set the title after loading data to facilitate testing
     svg.attr("title", "US Map")
 
     /* Definitions */
-    const defs = svg.append("defs")
-
-    const strokeShape = defs.append("filter").attr("id", "strokeShape")
-
-    strokeShape
-      .append("feMorphology")
-      .attr("operator", "erode")
-      .attr("radius", 1)
-      .attr("result", "erode")
-
-    strokeShape
-      .append("feGaussianBlur")
-      .attr("in", "erode")
-      .attr("stdDeviation", "3")
-      .attr("result", "blurFilter")
-
-    strokeShape
-      .append("feColorMatrix")
-      .attr("id", "colorMatrix2")
-      .attr("in", "blurFilter")
-      .attr("mode", "matrix")
-      .attr("values", "1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  0 0 0 14 -7")
-      .attr("result", "colorMatrix2")
-
-    strokeShape
-      .append("feGaussianBlur")
-      .attr("in", "erode")
-      .attr("stdDeviation", "2")
-      .attr("result", "blurFilter2")
-
-    strokeShape
-      .append("feComposite")
-      .attr("in", "colorMatrix2")
-      .attr("in2", "blurFilter2")
-      .attr("operator", "out")
-      .attr("result", "compositedStroke")
-
-    strokeShape
-      .append("feBlend")
-      .attr("in", "colorMatrix2")
-      .attr("in2", "compositedStroke")
-      .attr("mode", "multiply")
+    // const defs = svg.append("defs")
+
+    // const strokeShape = defs.append("filter").attr("id", "strokeShape")
+
+    // strokeShape
+    //   .append("feMorphology")
+    //   .attr("operator", "erode")
+    //   .attr("radius", 1)
+    //   .attr("result", "erode")
+
+    // strokeShape
+    //   .append("feGaussianBlur")
+    //   .attr("in", "erode")
+    //   .attr("stdDeviation", "3")
+    //   .attr("result", "blurFilter")
+
+    // strokeShape
+    //   .append("feColorMatrix")
+    //   .attr("id", "colorMatrix2")
+    //   .attr("in", "blurFilter")
+    //   .attr("mode", "matrix")
+    //   .attr("values", "1 0 0 0 0  0 1 0 0 0  0 0 1 0 0  0 0 0 14 -7")
+    //   .attr("result", "colorMatrix2")
+
+    // strokeShape
+    //   .append("feGaussianBlur")
+    //   .attr("in", "erode")
+    //   .attr("stdDeviation", "2")
+    //   .attr("result", "blurFilter2")
+
+    // strokeShape
+    //   .append("feComposite")
+    //   .attr("in", "colorMatrix2")
+    //   .attr("in2", "blurFilter2")
+    //   .attr("operator", "out")
+    //   .attr("result", "compositedStroke")
+
+    // strokeShape
+    //   .append("feBlend")
+    //   .attr("in", "colorMatrix2")
+    //   .attr("in2", "compositedStroke")
+    //   .attr("mode", "multiply")
```

Comment:
If we don't need this anymore we should remove it! Big comment critic here sorry :)

Reply:
We will need it again -- this is the styling for the map, but it obscures the actual outlines which makes building/fixing other things which interact with those outlines tough, so it's 'off' while in progress

Reply:
ok fo rme